example/35-namespacedcloudprofile.yaml

```diff
@@ -9,7 +9,10 @@ spec:
   parent:
     kind: CloudProfile
     name: local # could be any CloudProfile
-# kubernetes: # overwrite kubernetes versions if necessary
+  kubernetes:
```

Comment:
Can you augment this file with an example for overriding machine images (for provider-local)?